packages/aws-otel-collector/aws-otel-collector.service

```
@@ -0,0 +1,13 @@
+[Unit]
+Description=AWS OTEL collector
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/aws-otel-collector --config /etc/aws-otel-collector-default-config.yaml
```

I don't think the file should be named "default" (what if the values are not the default?) and the "config" part is redundant for files in `/etc`:
```suggestion
ExecStart=/usr/bin/aws-otel-collector --config /etc/aws-otel-collector.yaml
```

---

Ack, makes sense given we are going to make the config as bare as possible